### ISSUE TITLE
Feat: Clarify on_request space subcription mecanism.

### ISSUE THREAD

**Issue by inkhey** [tracim/tracim#3662]
## Feature description and goals

With the issue https://github.com/tracim/tracim/issues/3539 remains some unclear point that need to be clarified according to usage, etc..

## 1. Should we remove accepted subscription ?

https://github.com/tracim/tracim/issues/3539 spec say yes, but this probably mean losing some kind of information, that way in the current specification subscription are not dropped as this case remain unclear. Keeping them doesn't cause any damage, so this point was leave in this state now. This point also depend on the solution choose for 2 and 3

## 2. Where Should be the evaluator and evaluation date stored ?

In current implementation evaluator and evalution date are stored 2 time : 
- in subscription
- in tlm context info : author and creation date of the tlm (as the tlm store the subscription too, it store the information 2 time in fact in tlm)
Depending on the solution for 1 and 3, it can no longer make sense to keep both these fields in subscription table.

## 3. Should we give the evaluator and evaluation date to the subscriber ?

In current implementation, we do give this information by both tlm context info and workspace_subscription endpoint. If we do not want to give this information, we need to decide how. In a previous discussion a way to solve this may be : 
> - remove `evaluator` from `SubscriptionSchema`, making evaluator not visible from API
> - set `author` to empty for all `workspace_subscription` TLM.
see https://github.com/tracim/tracim/issues/3539#issuecomment-702128858

## Extra information 

see original discussion in ticket https://github.com/tracim/tracim/issues/3539

**Comment by grignards** [comment]
I do not have a strong opinion regarding the usefulness of keeping the subscriptions history, but changing it now would be costly as it would require a "not-so-small" refactoring of the work done in #3539. So I would rather keep it as we currently have implemented.

**Comment by inkhey** [comment]
> I do not have a strong opinion regarding the usefulness of keeping the subscriptions history, but changing it now would be costly as it would require a "not-so-small" refactoring of the work done in #3539. So I would rather keep it as we currently have implemented.

i agree, we can put this issue in tracimv3 backlog and adapt the frontend with current mecanism in https://github.com/tracim/tracim/issues/3587

**Comment by Skylsmoi** [comment]
This issue currently (v3.4.1) makes it impossible for a user to request an on demand space access after leaving it:
https://github.com/tracim/tracim/issues/3738
